kotlin-android/src/main/java/com/segment/analytics/kotlin/android/utilities/AndroidKVS.kt

```diff
@@ -0,0 +1,12 @@
+package com.segment.analytics.kotlin.android.utilities
+
+import android.content.SharedPreferences
+import com.segment.analytics.kotlin.core.utilities.KVS
+
+class AndroidKVS(val sharedPreferences: SharedPreferences): KVS {
+    override fun getInt(key: String, defaultVal: Int): Int =
```

Comment:
Should we consider a default parameter value for defaultVal?

Reply:
I noticed the rest of them were defaulted to 0

Reply:
this is an internal api, only being used in 1 place currently. im not sure if having default values will give us any additional benefits